app/scripts/proactive/model/Task.js

```diff
@@ -459,17 +461,23 @@ define(
 
             // documentation GI:  value will be displayed as link
             generateDocumentUrl: function(genericInformation) {
+              var hasDocumentation;
               var linkName = "Undefined";
               var documentationValue = "Undefined";
+
               if (this.attributes.hasOwnProperty('Generic Info') && this.attributes["Generic Info"] != "") {
                 for (var i in genericInformation) {
                   if (genericInformation[i]["Property Name"].toLowerCase() === 'task.documentation') {
+                    hasDocumentation = true;
                     linkName = genericInformation[i]["Property Value"];
                     documentationValue = genericInformation[i]["Property Value"];
                   }
                 }
-              } else {
-                console.log("No generic information task.documentation defined for this job");
+              } else{
+                console.log ("here");

```

Comment:
Maybe remove this logging?

Reply:
Thank you @luispinedamx, forgot to remove the debugging messages.